sirepo/package_data/static/js/srw.js

```diff
@@ -2548,9 +2548,9 @@ SIREPO.app.directive('simulationStatusPanel', function(appState, beamlineService
                 <div class="col-sm-12" data-simulation-status-timer="simState"></div>
                 <div data-ng-if="simState.showJobSettings()">
                   <div class="form-group form-group-sm">
-                    <div class="col-sm-12" data-model-field="\'jobRunMode\'" data-model-name="simState.model" data-label-size="6" data-field-size="6"></div>
-                    <div data-sbatch-options="simState"></div>
+                    <div data-model-field="\'jobRunMode\'" data-model-name="simState.model" data-label-size="6" data-field-size="6"></div>
```

Comment:
No need to escape the quotes inside a template literal - a bit easier to read without them I think